recipe/meta.yaml

```diff
@@ -9,8 +9,7 @@ source:
   sha256: 91dfe6f3f706ee8cc32d38edbbf304e9b7583fb37108fef38229617f8b3eba23
 
 build:
-  number: 0
-  skip: True  # [py>=36]
```

Comment:
I recall that @mingwandroid had a good reason to prevent the installation of this package in `Pythons >=3.6`